macrosAkka/src/main/scala/com/softwaremill/macwire/macwireakka/MacwireAkkaMacros.scala

```diff
@@ -0,0 +1,14 @@
+package com.softwaremill.macwire
+package macwireakka
```

Comment:
ah, better! :) just name it `akkasupport` to avoid problems with FSs & case-sensititity ;)